happytransformer/happy_text_to_text.py

```diff
@@ -6,18 +6,26 @@
 from transformers import Text2TextGenerationPipeline, AutoModelForSeq2SeqLM
 
 from happytransformer.happy_transformer import HappyTransformer
-from happytransformer.tt.trainer import TTTrainer
+from happytransformer.tt.trainer import TTrainer
```

Comment:
Wouldn't `TTTrainer` be more in line with the other types?

Reply:
I agree 
522ed3e